app/views/boards/_reorder.haml

```diff
@@ -0,0 +1,51 @@
+- reset_cycle
+#reorder-sections-table
+  .content-header
+    = content_for :reorder_title
+    #loading.float-right.hidden= loading_tag # TODO duplicate id
+    #saveerror.float-right.hidden
+      - # TODO duplicate id
+      = image_tag "icons/exclamation.png", title: 'Error', class: 'vmid', alt: '!'
```

Comment:
`alt`s should be changed in line with #1242 once that gets merged (in this case, it should be empty)

Reply:
Did get to go through merge conflict fun, yep xD